src/platforms/x11/input/input_platform.cpp

```diff
@@ -39,6 +39,18 @@
 #include <xkbcommon/xkbcommon.h>
 #include <xkbcommon/xkbcommon-x11.h>
 
+// xcb/xkb.h has a struct member named "explicit", which C++ does not like
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wkeyword-macro"
+#endif
+#define explicit explicit_
+#include <xcb/xkb.h>
+#undef explicit
+#ifdef __clang__
+#pragma clang diagnostic pop
+#endif
```

Comment:
I'm a little concerned that this incantation is repeated. But I'll let it go until there's a third occurance.